applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp

```diff
@@ -0,0 +1,56 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
+
+#include "custom_utilities/potential_flow_utilities.h"
+
+namespace Kratos
+{
+namespace PotentialFlowUtilities
+{
+template <int Dim, int NumNodes>
+array_1d<double, NumNodes> GetPotentialOnNormalElement(const Element& rElement)
+{
+    const int kutta = rElement.GetValue(KUTTA);
+    array_1d<double, NumNodes> potentials;
+
+    const Element::GeometryType& Geometry = rElement.GetGeometry();
```

Comment:
```suggestion
    const auto r_geometry = rElement.GetGeometry();
```

Reply:
Why is it better to use auto in this case?

Reply:
- shorter
- in this case it is clear what `GetGeometry` returns => I wouldn't use it for int/double, since ther things might go wrong
- imagine at some point you want this code to work with conditions too, then the compiler will determine the type

I have to admit that there are better examples but I think for such types it is a nice usecase